.test-infra/kafka/strimzi/02-kafka-persistent/README.md

```diff
@@ -25,4 +25,19 @@ https://github.com/strimzi/strimzi-kafka-operator/blob/main/examples/kafka/kafka
 The directories within [base](./base) contain the operator manifests and
 are named according to the version release.
 
-See [.test-infra/kafka/strimzi](../README.md) for requirements and usage.
+# Usage
+
+Simply deploy the cluster by using kustomize plugin of kubectl
+```
+kubectl apply -k .test-infra/kafka/strimzi/02-kafka-persistent
```

Comment:
I have two points:

1) When I tried this, I got the error:
```
error: unable to find one of 'kustomization.yaml', 'kustomization.yml' or 'Kustomization' in directory '.test-infra/kafka/strimzi/02-kafka-persistent'
```

This worked:
```
kubectl apply -k .test-infra/kafka/strimzi/02-kafka-persistent/overlays/gke-internal-load-balanced
```

2) Solution deployed into the default namespace. Was this intended? Original solution was in the default namespace. I don't mind either way.

Following specifies the namespace.

```
kubectl apply -k .test-infra/kafka/strimzi/02-kafka-persistent/overlays/gke-internal-load-balanced --namespace=strimzi
```

Reply:
Fixed the path.
Yeah it was supposed to be able to deploy to any namespace. I decided to put strimzi namespace into the instruction for the sake of completeness